addons/mass_mailing/data/mailing_data_templates.xml

```diff
@@ -91,22 +91,21 @@
     <template id="ab_testing_description" name="Mass Mailing: A/B Test Description">
         <div name="ab_testing_description" class="mb-2">
             <t t-if="mailing.ab_testing_completed">
-                <p t-if="mailing.ab_testing_pc == 100">
+                <p t-if="mailing.ab_testing_is_winner_mailing">
```

Comment:
can you add in your commit message that `ab_testing_pc` can be equal to 100% for all mailings because the domain of the recipient might be different ?